tool/jt.rb

```diff
@@ -1018,9 +1018,7 @@ def test_bundle(*args)
           end
           run({'GEM_HOME' => gem_home}, *["-S", "gem", "install", "bundler", "-v","1.14.6"])
           run({'GEM_HOME' => gem_home}, *["-S", "bundle", "install"])
-          # Need to workaround ruby_install name `jruby-truffle` in the gems binstubs (command can't be found )
-          # or figure out how to get it on the path to get `bundle exec rake` working
-          #run({'GEM_HOME' => gem_home}, *["-S", "bundle", "exec", "rake"])
+          run({ 'GEM_HOME' => gem_home }, *[File.join(gem_home, 'bin', 'bundle'), 'exec', File.join(gem_home, 'bin', 'rake')])
```

Comment:
`File.join` has a advantage of taking care of `//` when `GEM_HOME` ends with `/`.